CMakeLists.txt

```diff
@@ -196,11 +183,15 @@ IF(USE_TESTS)
         test/unittester.c
         test/utils_tests.c
         test/vector_tests.c
-        test/bip39_tests.c
-        test/bip44_tests.c
 )
-    TARGET_LINK_LIBRARIES(tests ${LIBDOGECOIN_NAME} )
+    TARGET_LINK_LIBRARIES(tests ${LIBDOGECOIN_NAME} ${LIBUNISTRING})
     ADD_TEST(NAME ${LIBDOGECOIN_NAME}_tests COMMAND tests)
+    # move test/wordlist to build
+    IF(WIN32)
+    ELSE()
+        execute_process(COMMAND mkdir -p ${PROJECT_SOURCE_DIR}/build/test/wordlist)
+        execute_process(COMMAND cp -r ${PROJECT_SOURCE_DIR}/test/wordlist ${PROJECT_SOURCE_DIR}/build/test)
+    ENDIF()
```

Comment:
Could I suggest a cmake based copy such as:

```
    FILE(GLOB_RECURSE WORDLISTS "${PROJECT_SOURCE_DIR}/test/wordlist/*.txt")
    FOREACH(wordlist ${WORDLISTS})
        GET_FILENAME_COMPONENT(wordlist_name ${wordlist} NAME)
        CONFIGURE_FILE(${wordlist} "${CMAKE_CURRENT_BINARY_DIR}/test/wordlist/${wordlist_name}" COPYONLY)
    ENDFOREACH()
```

just so you don't have to depend on platform specific command line stuff and the wordlists will always be in the binary directory, regardless of where that directory is

Reply:
Ah shoot, just saw this. I added another way to do this for WIN32 in #93.